Week6TicTacToe/TicTacToe.rb

```diff
@@ -0,0 +1,135 @@
+class TicTacToe
+  class ConsoleInputError < StandardError; end
+  attr_accessor :field
+
+  ARRAY_INDICES = %w[0 1 2].freeze
+
+  def initialize
+    @field = [
+      [254.chr, 254.chr, 254.chr],
+      [254.chr, 254.chr, 254.chr],
+      [254.chr, 254.chr, 254.chr]
+    ]
+  end
+
+  def play
+    console_line_interface
+  rescue ConsoleInputError => e
+    warn e.message
+  end
+
+  def console_line_interface
```

Comment:
very very long method. Please modify it

Reply:
i modified it by separating it into different methods for each functionality, please do check